io/serialport/25-serial.html

```diff
@@ -154,6 +154,7 @@ <h3>Outputs</h3>
             <td width="70px" data-i18n="serial.label.databits"></td>
             <td width="80px" data-i18n="serial.label.parity"></td>
             <td width="70px" data-i18n="serial.label.stopbits"></td>
+            <td></td>
```

Comment:
remove - not needed

Reply:
Yes it is, because all other cells on the row are fixed witdh. If we remove this flexible cell, the Baud rate cell will not align nicely.

Reply:
or all 4 can be in their own table equally spaced.